### DISCUSSION TITLE
Common environment variable across tasks

### DISCUSSION THREAD

**Discussion by jmccann** [concourse/concourse#8934] [Help & Support] [answered]
Is there a web or worker config that allows injecting environment variables in all ran tasks?

My use case is I'd like to disable telemetry for some tools we run in tasks by setting the related env but don't want to have to figure out which tasks to specifically add it to or worry about missing adding it to future created tasks.

**Comment by taylorsilva** [comment, marked as answer]
Sadly there is no way to globally inject env vars into every task container. Good use-case for adding such a feature though.

**Comment by cailloumajor** [comment]
Another use case I see is setting CI=true globally. Many tools recognize this variable to, e.g., prevent flooding the output.
